.github/workflows/docstring_coverage.yml

```
@@ -0,0 +1,42 @@
+  workflow_dispatch:
+  pull_request:
+
+jobs:
+  doc-coverage:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+
+      - name: Set Up Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.4)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)